populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
I get this error if I try to do a relative import  
SystemError: Parent module '' not loaded, cannot perform relative import
if you're not sure how to fix this that's fine and we can refactor the repo, but I would like to do that with a new groomed/planned story that we groom/plan with Simon to get input on the jenkins jobs. If you think we should block this PR on that we can mark the subtask as blocked. Thoughts?